src/test/java/controllers/fileSystem/FilePageTest.java

```diff
@@ -23,8 +23,8 @@
 
     @Before
     public void createInstance() {
-        fileSystem = new FileSystem();
-        fileSystemStub = new FileSystemStub();
+        fileSystem = new FileSystem("/Users/gabi/Desktop/MyApprenticeship/IPM/Java/HTTPServer/cob_spec/public");
```

Comment:
THIS does not exist on travis! Can we create a temporary directory or something? Maybe refernece a file that exists within the very same git repository?